### PR TITLE
Enable Layout/ArgumentAlignment cop in RuboCop

### PR DESCRIPTION
**Pull request by tnir**
Enable `Layout/ArgumentAlignment` cop in RuboCop.

Partially updates #672

Signed-off-by: Takuya Noguchi [takninnovationresearch@gmail.com](https://github.com/sponsors/tnir)

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 AllCops:
   DisabledByDefault: true
+  Exclude:
+    - "node_modules/**/*"
+    - "tmp/**/*"
+    - "vendor/**/*"
+    - ".git/**/*"
+    - "bin/*"
```
Comment (tnir) [inline .rubocop.yml:8]: Just want to exclude binstubs in "bin/*", but the default is preset: https://github.com/rubocop/rubocop/blob/v1.31.2/config/default.yml#L63-L67

Without it, we will see the following offenses:
```ruby
$ bundle exec rubocop
Inspecting 49 files
..........C...C...C..............................

Offenses:

bin/middleman:12:3: C: [Correctable] Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.
  Pathname.new(__FILE__).realpath)
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
bin/puma:12:3: C: [Correctable] Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.
  Pathname.new(__FILE__).realpath)
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
bin/rake:11:3: C: [Correctable] Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.
  Pathname.new(__FILE__).realpath)
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

49 files inspected, 3 offenses detected, 3 offenses autocorrectable
```

Comment (deivid-rodriguez) [reply .rubocop.yml:8]: I think you can fix that by adding

```yaml 
inherit_mode:
  merge:
    - Exclude
```

Comment (tnir) [reply .rubocop.yml]: Oh, I see!

**Review by deivid-rodriguez — APPROVED** [review]
Actually my favorite style is `with_fixed_indentation`, but I'm happy with anything that's consistent.

**Comment by tnir** [comment]
Got it. In this repo, we have just 2 occurrences only in `config.rb`, Middleman configuration DSL. 🙏

**Pull request merged**